config/kubernetes/demo/ingress-live.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     external-dns.alpha.kubernetes.io/set-identifier: track-a-query-ingress-track-a-query-demo-green
     external-dns.alpha.kubernetes.io/aws-weight: "100" 
```

Comment:
Question about this - should it be zero?

Reply:
no 100 is correct